crates/rpc-proto/proto/account.proto

```diff
@@ -3,30 +3,48 @@ package account;
 
 import "digest.proto";
 
+// An account ID.
+//
+// A Miden account ID is a 120-bit value derived from the commitments to account code and storage,
+// and a random user-provided seed.
```

Comment:
In the long run we should probably link to official documentation site which would specify the account ID:

```suggestion
// Uniquely identifies a specific account.
//
// A Miden account ID is a 120-bit value derived from the commitments to account code and storage,
// and a random user-provided seed.
```